docs/job.md

```diff
@@ -145,37 +147,82 @@ The overall job state can be added to the job list response:
 
 View details for a single job. The file listing format is described in [compendium files](compendium/files.md)
 
-`curl https://…/api/v1/job/$ID`
+`curl https://…/api/v1/job/$ID?details=all`
 
-`GET /api/v1/job/:id`
+`GET /api/v1/job/:id?details=all`
 
 ```json
 200 OK
 
 {
-  "id":"nkm4L",
-  "compendium_id":"a4Dnm",
-  "creation_date": Date,
+  "id": "nkm4L",
+  "compendium_id": "a4Dnm",
+  "user": "0000-0001-9757-xxxx",
```

Comment:
Please use the o2r user ORCID for examples: https://orcid.org/0000-0001-6021-1617